src/messageHandler.ts

```diff
@@ -0,0 +1,22 @@
+import { Client, Message } from "discord.js";
+const BOT_ID = "863633669989466142"
+// Refer to https://github.com/discordjs/discord.js/blob/master/typings/index.d.ts for DiscordJS typings
+
+export interface IMessageHandler {
```

Comment:
docs here to explain the interface and subclasses that implement the interface may be helpful